scripts/build/tests.mjs

```diff
@@ -82,6 +83,13 @@ export async function runConsoleTests(runJs, defaultReporter, runInParallel, opt
     /** @type {string[]} */
     const args = [];
 
+    // enable code coverage using 'c8'
+    let execPath = process.execPath;
+    if (coverage) {
+        args.push("exec", "c8", "--clean", execPath);

```

Comment:
It's easier to continue using `npm exec` to avoid having to deal with trying to spawn a local package bin and deal with multiple platforms myself, though I can add `--prefer-offline` or `--offline` to have it prefer the local version.

Reply:
To clarify, I am both continuing to use `npm exec` for convenience and *also* adding `c8` to `devDependencies`.

Reply:
> It's easier to continue using `npm exec` to avoid having to deal with trying to spawn a local package bin and deal with multiple platforms myself, though I can add `--prefer-offline` or `--offline` to have it prefer the local version.

What do you mean by platforms? It should be at `node_modules/c8/bin/c8.js`; we execute eslint and mocha using similar methods.